toolchain/openapi/generateApiDocs.py

```diff
@@ -26,7 +26,7 @@
 	"openapi": "3.1.0",
 	"info": {
 		"description": "ArangoDB REST API Interface",
-		"version": "1.0",
+		"version": "3.10.5",
```

Comment:
The version should not be hard-coded, but we don't have something like https://github.com/arangodb/docs/blob/main/_data/versions.yml (yet)